flink-kubernetes-webhook/pom.xml

```diff
@@ -61,41 +62,4 @@ under the License.
         </dependency>
     </dependencies>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-shade-plugin</artifactId>
```

Comment:
I think we probably need to keep the shade plugin to include the netty classes in our jar. 

But we can definitely remove the `flink-core` dependency and the exclusions for the `flink-kubernetes-operator`